clang/test/Frontend/arm-branch-protection-lto.c

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: arm-registered-target
+
+// RUN: %clang -flto -target thumbv7m-unknown-unknown-eabi -mbranch-protection=pac-ret %s -S -o - 2>&1 | FileCheck %s
```

Comment:
prefer  `--target=` for new tests

For codegen tests we prefer `%clang_cc1` to `%clang`

Reply:
Done